sdk/textanalytics/azure-ai-textanalytics/pom.xml

```diff
@@ -36,7 +36,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.6.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.7.0-beta.2</version> <!-- {x-version-update;beta_com.azure:azure-core;dependency} -->
```

Comment:
We shouldn't have these beta updates?

Reply:
we have to actually. I also need to change the identity version. Otherwise Test from source will fail.

Reply:
This is reverted. The only places that need this change are event hubs, and spring